### DISCUSSION TITLE
听歌系列之蔡琴 | /cn/2021/07/tsai-chin/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1430] [Comments]
https://yihui.org/cn/2021/07/tsai-chin/

[a comment by an automated account is not shown]

**Comment by licw58** [comment]
> 温柔总是冷里好。
> 钩月雪沙，千年两相照。
> 风去烟尘旧梦皎，梦植心田荒漠茂。
> 
> 多情不若泣中笑。
> 天涯易随，笑别成全少。
> 漠撒泪花无人晓，回首深埋来时道。

第一次读到这首词时，总觉得哪里需要改动一下，昨天偶尔想起来，想起了最后一句，你看看如何？纯属为了格律和对仗，硬凑场景。

> 温柔总是冷里好。
> 钩月雪沙，千年两相照。
> 风去烟尘旧梦皎，梦植心田荒漠茂。
> 
> 多情不若泣中笑。
> 天涯易随，笑别成全少。
> 漠撒泪花无人晓，人寄枫林清秋皓。

**Reply by yihui** [reply]
嗯，对仗非常好！也扭转了原词的基调，显得更淡远和从容。
